src/Tools/DynamoFeatureFlags/FeatureFlagsClient.cs

```diff
@@ -86,7 +86,9 @@ internal FeatureFlagsClient(string userkey, string mobileKey = null, bool testMo
                 AllFlags = LdValue.ObjectFrom(new Dictionary<string,LdValue> { { "TestFlag1",LdValue.Of(true) },
                     { "TestFlag2", LdValue.Of("I am a string") },
                     //in tests we want instancing on so we can test it.
-                    { "graphics-primitive-instancing", LdValue.Of(true) } });
+                    { "graphics-primitive-instancing", LdValue.Of(true) },
+                    //in tests we want search debouncing on so we can test it.
+                    { "searchbar_debounce", LdValue.Of(true) } });
```

Comment:
do we really need a feature flag for a text box debouncer ?

Reply:
I've outlined why I think we should use a feature flag for this specifically in the other PR.
https://github.com/DynamoDS/Dynamo/pull/15733#issuecomment-2557291014
fight me⚔️ 

but in general, we should get into the habit of using them everywhere.

Reply:
My opinion:
I would agree to use more fflags for most new features or significant changes in behavior. But for performance improvements (especially ones that have little impact area) I think they could just be turned on.

Reply:
I am fine with having the flags for this. Thinking more about it, it is kind of a critical feature (in canvas node search)